packages/navigation-menu/src/lib/navigation-menu/components/sub-menu/sub-menu.component.scss

```diff
@@ -25,6 +25,10 @@
   }
 }
 
+.a-back-link {
```

Comment:
This seems incorrect. This class doesn't provide the black styling I would expect.
Maybe use something like:
```css
.u-cursor-pointer {
  cursor: pointer;
}
```

Reply:
The default font weight is on purpose, isn't it @TriangleJuice  ?

Reply:
Is it possible that you're discussing two different things here? It is true however that a `.u-cursor-pointer` utility class would be a better choice.